docs/docs/faq.md

```diff
@@ -25,6 +25,16 @@ On the other hand, you can still add whatever you want as a middleware which wra
 
 As of now, No. But it should not be a big issue because the reverse-proxy which gRPC-Gateway generates usually works as an independent process and communicates with your gRPC server over TCP or a Unix domain sockets (Unix systems only).
 
+If you do insist that your gateway should be written in the same language
+as your server, there are a few projects that will help with this:
+
+* NodeJS:
+  * [grps-dynamic-gateway](https://github.com/konsumer/grpc-dynamic-gateway)
+* TypeScript / Javascript
+  * Not a direct analog, but [rest2grpc](https://www.npmjs.com/package/rest2grpc) is similar.
```

Comment:
Could we add the following too:
- Google Cloud Platform HTTP/JSON gRPC transcoding: https://cloud.google.com/endpoints/docs/grpc/transcoding
- Envoy proxy gRPC-JSON transcoder: https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_filters/grpc_json_transcoder_filter

Reply:
Done, please let me know what do you think.